### ISSUE TITLE
Docker images are huge

### ISSUE THREAD

**Issue by weholt** [kennethreitz/responder#159]
I tried using the docker image suggested in the documentation and the resulting image was 1.37GB in size. Then I used the standard python:3.6 and the size came down to 982MB, but I suspect it's possible to get that size down if we can use an alpine based image. Is that possible or does responder or one of its dependencies require some library or compiler etc, not present/available in alpine based images?

**Comment by kennethreitz** [comment]
manylinux wheels are not supported on alpine.

**Comment by benregn** [comment]
@weholt you could look at using `python:3.6-slim` or use Docker multi-step builds. If you have any questions, just ask.

**Comment by weholt** [comment]
Found out I needed gcc, build-essentials etc to build the dependencies in my images when I used the slim version, but got it working after a few tries! Using the slim version and the following Dockerfile gave me a finished image of just 199MB! 

FROM python:3.6-slim as builder

RUN apt-get update
ENV BUILD_DEPS='g++ gcc git make build-essential'
RUN apt-get install -y $BUILD_DEPS --no-install-recommends
RUN apt-get install build-essential
RUN pip install --upgrade pip

COPY . /app
RUN pip install responder

FROM python:3.6-slim
COPY --from=builder /root/.cache /root/.cache
COPY --from=builder /app /app
RUN pip install --upgrade pip
RUN pip install responder

I see the docs use pipenv to install responder but I got an error doing that so I tried using pip instead without any issues. Can provide details if wanted on the build errors. Anyhow - happy with the size of my images. Thanks!
